### PR TITLE
Add Docker-hub documentation

### PR DESCRIPTION
**Pull request by f-necas**
To fill all docker hub documentation 

Main PR :
- https://github.com/georchestra/georchestra/pull/4135

### CODE REVIEW

File: DOCKER_HUB.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Quick reference
+
+-    **Maintained by**:  
+      [georchestra.org](https://www.georchestra.org/)
+
+-    **Where to get help**:  
+     the [Georchestra Github repo](https://github.com/georchestra/georchestra), [Matrix chat](https://matrix.to/#/#georchestra:libera.chat), Stack Overflow
```
Comment (Gaetanbrl) [inline DOCKER_HUB.md:7]: Matrix chat is not the native (official ?) chat. 
Maybe we can insert https://kiwiirc.com/nextclient/irc.libera.chat/georchestra link ?

Comment (edevosc2c) [reply DOCKER_HUB.md:7]: I think we can link both matrix and IRC. Both are interconnected between each other.

**Review by edevosc2c — APPROVED** [review]
Apart from gaetan comment, good for me.

File: DOCKER_HUB.md (hunk 1 above)
Comment (Gaetanbrl) [reply DOCKER_HUB.md:7]: > Both are interconnected between each other.

I guess this is not transparent for everyone. Newcomers have to know that both exists and I suggest to be consistent with georchestra web site (at least show same informations - IRC)

https://www.georchestra.org/fr/communaute.html

image



**Clearly, this is a detail.**

Comment (edevosc2c) [reply DOCKER_HUB.md]: ok maybe we can only link IRC for the moment but latter on propose matrix too on official georchestra website then changing the docker hub description.

**Pull request merged**